code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/pandora.dm

```diff
@@ -125,7 +125,7 @@
 			new /obj/effect/temp_visual/hierophant/blast/pandora(t, src)
 
 /mob/living/simple_animal/hostile/asteroid/elite/pandora/proc/pandora_teleport(target)
-	ranged_cooldown = world.time + cooldown_time
+	ranged_cooldown = world.time + (cooldown_time * 3)
```

Comment:
Not where you should be altering cooldown time. Just change it from 20 to 60 where it's defined

Reply:
That would affect the cooldown of all attacks